src/components/composed/LockedBalanceList/LockedBalanceListItem.js

```diff
@@ -0,0 +1,24 @@
+import { format } from 'date-fns'
+
+import './LockedBalanceListItem.css'
+
+const LockedBalanceListItem = ({ index, utxo }) => {
+  return (
+    <tr
+      key={index}
+      style={{
+        backgroundColor: index % 2 === 0 ? 'white' : 'rgb(247, 250, 250)',
+      }}
+    >
+      <td style={{ padding: '10px' }}>
+        {format(
+          new Date(utxo.utxo.lock.content.timestamp * 1000),
```

Comment:
This covers lock by date. We also need to deal with Lock by block count. To recreate this look to wallets with recently withdrawed coins from delegation. Or just make new delegation and withdraw.
In explorer we have next snippet:
```javascript
if (data_utxos[i].utxo.type === "LockThenTransfer") {
      if (data_utxos[i].utxo.lock.type === "UntilTime" && data_utxos[i].utxo.lock.content.timestamp > Date.now() / 1000) {
        vesting.push({
          coin: "ML",
          symbol: "ML",
          type: "Coin",
          amount: data_utxos[i].utxo.value.amount.decimal,
          unlock_time: data_utxos[i].utxo.lock.content.timestamp,
        });
      }
      if (data_utxos[i].utxo.lock.type === "ForBlockCount") {
        const res_tx = await fetch(NODE_API_URL + "/transaction/" + data_utxos[i].outpoint.source_id, {
          cache: "no-store",
          headers: {
            "Content-Type": "application/json",
          },
        });
        const data_tx = await res_tx.json();

        const res_block = await fetch(NODE_API_URL + "/block/" + data_tx.block_id, {
          cache: "no-store",
          headers: {
            "Content-Type": "application/json",
          },
        });
        const data_block = await res_block.json();

        if (data_utxos[i].utxo.lock.content + data_block.height > block_height) {
          vesting.push({
            coin: "ML",
            symbol: "ML",
            type: "Coin",
            amount: data_utxos[i].utxo.value.amount.decimal,
            unlock_time: data_block.header.timestamp.timestamp + data_utxos[i].utxo.lock.ForBlockCount * 60 * 2,
            unlock_block: data_utxos[i].utxo.lock.ForBlockCount + data_block.height,
          });
        }
      }
    }
```

res_tx no need to fetch from network it already should be in wallet state